### DISCUSSION TITLE
autobreak during landing

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#3812] [General]
Hi simmers,

I used to fly the custom FBW mod. Could it be possible, that the autobreak during landing does not have any effect? I didn´t regognize a decceleration, nor with low or med breaks?! I have the Airbus quadrant from Thrustmaster. There is a pseudo knob for the autobreak (more Boeing than Airbus :-p). Could this be the posible problem? 

greets Klaus

**Comment by ChruutvoLuzi** [comment]
Works fine on my side with the Experimental Versions. Btw: The Autobrake Knob on the TCA Quadrant is an Airbus one, but from an A380 :D

**Reply by CptBayer** [reply]
The autobreak during landing did only work, when the knob on the quadrant is set to the belonging button in the cockpit. After the last update of the custom FBW, this method doesn´t have any effect?! No decel after touchdown showed on the buttons nor a regognition of decel at all.

Any suggestions ?

Greets Klaus

**Reply by CptBayer** [reply]
working again .... don´t know what happend ...

cheers ...

**Reply by CptBayer** [reply]
Hi simmers and developers,


afteer the last update autobreaks aren´t working again. No decel on the button and no decel of the aircraft at all. As written above,I am using the TCA quadrant. I tested both options with knob of the quadrant in the correspondening postion and without ... Well, therefore are nice ECAM messages displayed with hot breaks :-) ...

greets klaus
